lib/rubocop/cop/rspec/repeated_example_group_description.rb

```diff
@@ -0,0 +1,98 @@
+# frozen_string_literal: true
```

Comment:
Are there any worth freezing? :D

Reply:
Configured my editor to auto fix offenses, so this line is added automatically. And I see this line everywhere in the source code. And yes, there is `MSG` string literal, so maybe it is still worth to freeze it :)